ontology/yaml/resources/fields/telemetry_fields.yaml

```diff
@@ -831,7 +834,23 @@ literals:
 - fire_alarm:
   - ACTIVE
   - INACTIVE
-
+  
+- valve_command:
+  - OPEN
+  - CLOSED
+- valve_status:
+  - OPEN
+  - CLOSED
+- low_battery_alarm:
+  - ACTIVE
+  - INACTIVE
+- load_bypass_mode:
```

Comment:
i did not see this used elsewhere in the PR. can you please tell me where it is used? any unused fields should be omitted.

Reply:
oops, this will be for next pull req, sorry. removed.